app/scripts/migrations/068.js

```diff
@@ -0,0 +1,159 @@
+import { cloneDeep } from 'lodash';
+
+const version = 68;
+
+/**
+ * Transforms the PermissionsController and PermissionsMetadata substates
+ * to match the new permission system.
+ */
+export default {
+  version,
+  async migrate(originalVersionedData) {
+    const versionedData = cloneDeep(originalVersionedData);
+    versionedData.meta.version = version;
+    const state = versionedData.data;
+    const newState = transformState(state);
+    versionedData.data = newState;
+    return versionedData;
+  },
+};
+
+function transformState(state) {
+  const { PermissionsController = {}, PermissionsMetadata = {} } = state;
+
+  const {
+    domainMetadata = {},
+    permissionsHistory = {},
+    permissionsLog = [],
+  } = PermissionsMetadata;
+
+  return {
+    ...state,
```

Comment:
Aren't `PermissionsController` and `PermissionsMetadata` still in this state? This is returning the old state alongside the new state, rather than replacing the old with the new.

Reply:
This was fixed as of e48e20a